internal/pkg/cli/push.go

```diff
@@ -87,9 +79,19 @@ func pushCommand(root *rootCommand) *cobra.Command {
 				dryRun := root.options.GetBool("dry-run")
 				if dryRun {
 					logger.Info("Dry run, nothing changed.")
+					logger.Info(`Push done.`)
 					return nil
 				}
 
+				// Send cmd successful/failed event
+				if eventSender, err := root.GetEventSender(); err == nil {
+					defer func() {
+						eventSender.SendCmdEvent(root.start, cmdErr, "push")
```

Comment:
Namiesto 2 prikazov uz iba jeden.
- `defer` tj. vykonaj na konci
- obalene do anonymnej funkcie, pretoze nas zaujimaju hodnoty v case vykonania, nie v case definicie (zavolania `defer`)
- ak prikaz prejde, tak `cmdErr == nil`, ... je to definovane ako navratova hodnota funkcie o uroven vyssie
- ak prikaz nejprejde, tak sa vola `return err` a `cmdErr = Error ....`
- ak nastane panic, tak to detekuje v `SendCmdEvent` a tiez sa posle, ze prikaz zlyhal.

Reply:
Tj. funguje to rovnako, ale staci jeden prikaz.